code/modules/food_and_drinks/food/snacks/meat.dm

```diff
@@ -86,7 +86,14 @@
 	filling_color = "#6B8E23"
 	tastes = list("meat" = 4, "scales" = 1)
 	foodtype = MEAT | RAW
-
+// yogs start -- gorilla people
+/obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/gorilla
+	icon_state = "gorillameat"
+	desc = "Damn dirty steak."
+	filling_color = "#FF0000"
+	tastes = list("meat" = 4, "fur" = 1)
+	foodtype = MEAT | RAW
+// yogs end
```

Comment:
Can you dump this class definition into ``/yogstation/code/modules/food_and_drinks/food/snacks/meat.dm``?